youtube_dl/extractor/nytimes.py

```diff
@@ -96,6 +98,43 @@ def get_file_size(file_size):
             'thumbnails': thumbnails,
         }
 
+    def _extract_podcast_from_json(self, json, page_id, webpage):
+        audio_data = self._parse_json(json, page_id, transform_source=js_to_json)['data']
+        
+        description = audio_data['track']['description']
+        if not description:
+            description = self._html_search_meta(['og:description', 'twitter:description'], webpage)
+
+        episode_title = audio_data['track']['title'].strip("‘’") # strip curlyquotes
```

Comment:
In this case curlyquotes seem to delimit the field for no apparent reason, and having quotes of any kind in the filesystem is quite a hassle, so stripping them seems warranted:

http://www.nytimes.com/2016/10/14/podcasts/revelations-from-the-final-weeks.html

```
var NYTD=NYTD || {}; NYTD.FlexTypes = NYTD.FlexTypes || []; NYTD.FlexTypes.push({
    "target":"FT100000004709062",
    "type":"NYT5-Podcast-Audio",
    "data":{
	"podcast":{
	    "title":"The Run-Up",
	    "landing":"",
	    "episode":"Episode 20",
	    "episodeurl":"http:\/\/www.nytimes.com\/2016\/10\/11\/podcasts\/on-sex-and-power-in-the-campaign-from-the-women-who-cover-it.html",
	    "subscribe":[{
		"name":"iTunes",
		"url":"https:\/\/geo.itunes.apple.com\/us\/podcast\/the-run-up\/id1142083165?mt=2"},
			 {
			     "name":"Google Play Music",
			     "url":"https:\/\/play.google.com\/music\/listen?authuser&u=0#\/ps\/Iotf2cvoqfvgkt4ywtmpdb3gjlu"}]},
	"track":{
	    "title":"‘He Was Like an Octopus’",
	    "description":"",
	    "credit":"",
	    "source":"https:\/\/rss.art19.com\/episodes\/b0684887-7f4b-4bae-a7af-de2e84d8f4f7.mp3",
	    "duration":2130}
    }});
```